build.gradle

```diff
@@ -369,6 +369,17 @@ subprojects {
     configProperties = checkstyleConfigProperties("import-control.xml")
     toolVersion = '8.10'
   }
+
+  configure(checkstyleMain) {
+    group = 'Verification'
+    description = 'Run checkstyle on all main Java sources'
+  }
+
+  configure(checkstyleTest) {
+    group = 'Verification'
+    description = 'Run checkstyle on all test Java sources'
+  }
```

Comment:
Out of curiosity, what does this do? In general, it's good practice to explain the reason for unrelated changes in the PR description.

Reply:
Setting a group + description on a task will add it to the `gradle tasks` output. Without it, you have to search through the gradle file to discover what tasks exist. 

```
$ gradle tasks

...

Verification tasks
------------------
check - Runs all checks.
checkScoverage
checkstyleMain - Run checkstyle on all main Java sources
checkstyleTest - Run checkstyle on all test Java sources
jacocoTestCoverageVerification - Verifies code coverage metrics based on specified rules for the test task.
jacocoTestReport - Generates code coverage report for the test task.
reportScoverage
spotlessApply - Applies code formatting steps to sourcecode in-place.
spotlessCheck - Checks that sourcecode satisfies formatting steps.
test - Runs the unit tests.
testScoverage


BUILD SUCCESSFUL in 7s
1 actionable task: 1 executed
```

Reply:
And agreed, I should have added a note in the PR description.

Reply:
Thanks for the explanation!